CMakeLists.txt

```diff
@@ -43,7 +43,11 @@ function(add_subdirectory_exclude_platforms NAME)
 endfunction()
 
 # Add blink example
-add_subdirectory_exclude_platforms(blink)
+if (PICO_CYW43_SUPPORTED AND NOT TARGET pico_cyw43_arch)
+    message("Wi-Fi driver is not available")
```

Comment:
I had meant "Skipping blink example as Wi-Fi driver is not available"

Can you also move this check into the blink CMakeLists.txt

Reply:
> Can you also move this check into the blink CMakeLists.txt

By the time we're "in" the blink CMakeLists.txt, is it not then too late to call `add_subdirectory_exclude_platforms(blink)` ?